gprofiler/profilers/python_ebpf.py

```diff
@@ -0,0 +1,251 @@
+import glob
+import os
+import signal
+from pathlib import Path
+from subprocess import Popen
+from threading import Event
+from typing import Dict, List, NoReturn, Optional
+
+from gprofiler.exceptions import (
+        CalledProcessError,
+        StopEventSetException,
+)
+from gprofiler.gprofiler_types import (
+        ProcessToProfileData,
+        ProfileData,
+)
+from gprofiler.profilers.profiler_base import ProfilerBase
+from gprofiler.utils import (
+        poll_process,
+        random_prefix,
+        resource_path,
+        run_process,
+        start_process,
+        wait_event,
+        wait_for_file_by_prefix,
+)
+
+from gprofiler import is_windows
+if not is_windows():
```

Comment:
Instead of handling windows in this file - it is better if we refrain from importing it, PyPerf will not run on Windows as it's ebpf. Let's remove `is_windows` code from here and instead focus on avoiding to import the file

Reply:
Done